library/oneview_logical_interconnect.py

```diff
@@ -98,7 +107,10 @@
 
 - name: Update the interconnect settings
```

Comment:
Its better to mention here also this functionality is supported only till Oneview3.1

Reply:
Sijeesh has suggested to keep it in examples only. So we will be keeping it at one place only.